commons-test/src/main/java/org/infinispan/commons/test/CommonsTestBlockHoundIntegration.java

```diff
@@ -19,7 +19,7 @@ public void applyTo(BlockHound.Builder builder) {
             // TestNG may be started on main thread and load this before renaming to testng
             t.getName().startsWith("main") ||
             // The threads our tests run on directly
-            t.getName().startsWith("testng") ||
+            t.getName().startsWith("TestNG") ||
```

Comment:
I believe we want both lower and upper case. I know when running the test the thread gets renamed to testng-<name>.